netbox_librenms_plugin/templates/netbox_librenms_plugin/_interface_sync_content.html

```
@@ -303,23 +346,32 @@ <h5 class="modal-title" id="netboxOnlyInterfacesModalLabel">
                         <table class="table table-hover">
                             <thead>
                                 <tr>
+                                    {# In migrated mode the bulk-delete button is hidden, so the #}
+                                    {# select-all column is a dead control — drop it entirely. #}
+                                    {% if not migrated_to_marker %}
                                     <th>
                                         <input type="checkbox" id="select-all-netbox-interfaces"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)